### PR TITLE
move custom-tools to NFS

### PR DESCRIPTION
**Pull request by gmauro**

### CODE REVIEW

**Comment by erasche** [comment]
If it isn't too much trouble, could you also make `files/galaxy-test/config/tool_conf.xml` into a template and use that variable there?

**Comment by erasche** [comment]
Otherwise :+1:

**Comment by gmauro** [comment]
not sure about model_tools_path var
  https://github.com/usegalaxy-eu/infrastructure-playbook/blob/22ee2f3ee36bfa14697a89c8cba60c249c8126bd/files/galaxy-test/config/tool_conf.xml#L39

File: group_vars/galaxy-test.yml [hunk 1]
```diff
@@ -648,3 +648,5 @@ galaxy_config_templates:
     #dest: "{{ galaxy_config_dir }}/grt.yml"
   - src: "{{ galaxy_config_template_src_dir }}/config/pulsar_app.yml"
     dest: "{{ galaxy_config_dir }}/pulsar_app.yml"
+  - src: "{{ galaxy_config_template_src_dir }}/config/tool_conf.xml.j2"
+    dest: "{{ galaxy_config_dir }}/tool_conf.xml"
```
Comment (erasche) [inline group_vars/galaxy-test.yml]: I think it also needs to be removed from the galaxy_config_files var

**Comment by erasche** [comment]
It's a built in galaxy thing, should be fine to leave it alone. Thanks for doing this!

**Pull request merged**